lua/nvim-gps/init.lua

```diff
@@ -5,60 +5,47 @@ local gps_utils = require("nvim-gps.utils")
 
 local M = {}
 
-local config = {
+-- Default configuration that can be overridden by users
+local default_config = {
+	enabled = true,
```

Comment:
What does this enabled do?

Reply:
This works like a placeholder where the user can disable the plugin for a language. This ensures that by default all languages are enabled in the configuration level.